docs/src/pinn/debugging.md

```diff
@@ -52,9 +54,9 @@ dim = length(domains)
 dx = 0.1
 strategy = NeuralPDE.GridTraining(dx)
 
-_pde_loss_function = NeuralPDE.build_loss_function(eq,indvars,depvars,phi, derivative,chain,initθ,strategy)
+_pde_loss_function = NeuralPDE.build_loss_function(eq,indvars,depvars,phi, derivative,chain,initθ,eltypeθ,strategy)
```

Comment:
could it be less breaking to just `eltype(inittheta)` inside of the function?

Reply:
yep, it makes sense. also we can pass only `chain` because initθ = DiffEqFlux.initial_params(chain)